lib/watcher/heart-beat.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+exports.worker = function(handle) {
+  var intervalObj = null;
+  var _heartBeatInterval = 15 * 1000;
+  intervalObj = setInterval(function() {
+    handle.send({
+      cmd: 'metrics',
+      record: JSON.stringify({dummyRecord: ''}),
+    });
+  }, _heartBeatInterval);
+
+  this.stopWorker = function() {
```

Comment:
`stop()` - why put the name "worker" in a method on a Worker object.

Also, `this` is a strange way of referring to the module global, its the same as `exports` here, because exports.worker is not being called as a constructor, but you can't tell that from this code! Use `exports`, its clearer.